src/main/steps/home/get.ts

```diff
@@ -1,19 +1,21 @@
 import { Response } from 'express';
 
-import { CaseApi } from '../../app/case/CaseApi';
 import { AppRequest } from '../../app/controller/AppRequest';
+import { Form } from '../../app/form/Form';
 import { CHECK_ANSWERS_URL, YOUR_DETAILS_URL } from '../../steps/urls';
+import { form as firstQuestionFormContent } from '../your-details/content';
 
 export class HomeGetController {
   public get(req: AppRequest, res: Response): void {
     if (req.session.userCase.divorceOrDissolution !== res.locals.serviceType) {
       throw new Error('Invalid case type');
     }
 
-    const isCasePartiallyComplete = Object.entries(req.session.userCase).some(
-      ([key, value]) => !CaseApi.SPECIAL_FIELDS.includes(key) && value
-    );
+    const isFirstQuestionComplete = () => {
```

Comment:
why is this a function?

Reply:
i think its to do with how a new user will be redirected to the check your answers page as when they sign up, the name and lastname are already there so it doesnt go to the first question as case is partially complete.

Reply:
👍 Good point, I've remove that as it's unnecessary